src/routes/console/project-[project]/auth/templates/strote.ts

```diff
@@ -0,0 +1,33 @@
+import type { Models } from '@appwrite.io/console';
+import { writable } from 'svelte/store';
+
+export const emailTemplate = writable<Models.EmailTemplate>({
+    type: null,
+    locale: null,
+    message: null,
+    senderName: '',
+    senderEmail: null,
+    replyTo: null,
+    subject: null
+});
+
+export const baseEmailTemplate = writable<Models.EmailTemplate>({
+    type: null,
+    locale: null,
+    message: null,
+    senderName: null,
+    senderEmail: null,
+    replyTo: null,
+    subject: null
+});
```

Comment:
why are we extending the base models here? Aren't these supposed to be part of the SDK itself?

Reply:
We are initializing the store with null values to avoid error, the model is unchanged

Reply:
Ah got it 👍